packages/terra-status-view/docs/DEPENDENCIES.md

```diff
@@ -0,0 +1,20 @@
+# Dependency Information
```

Comment:
The scripts were updated so that this should only be getting generated on publish. You should remove it from the PR.